drivers/sensor/bosch/bme680/bme680.c

```diff
@@ -187,8 +187,8 @@ static uint8_t bme680_calc_res_heat(struct bme680_data *data, uint16_t heatr_tem
 					   * heatr_temp * 5) / 100)
 					 + 3276800) / 10);
 	var3 = var1 + (var2 / 2);
-	var4 = (var3 / (data->res_heat_range + 4));
-	var5 = (131 * data->res_heat_val) + 65536;
+	var4 = (var3 / (((uint8_t)data->res_heat_range) + 4));
+	var5 = (131 * ((int8_t)data->res_heat_val)) + 65536;
```

Comment:
No division here. Maybe the line below?

Reply:
Yeah - Essentially the complaint is on the following areas:
https://github.com/zephyrproject-rtos/zephyr/blob/f41cb40a6e746c894e6126a7dee9abeb3814d49f/drivers/sensor/bosch/bme680/bme680.c#L192
https://github.com/zephyrproject-rtos/zephyr/blob/f41cb40a6e746c894e6126a7dee9abeb3814d49f/drivers/sensor/bosch/bme680/bme680.c#L190

In the first case, we're addressing where `var5` comes from.